doc/manual.rst

```diff
@@ -0,0 +1,116 @@
+.. Copyright (c) 2008-2020 OpenShot Studios, LLC
+ (http://www.openshotstudios.com). This file is part of
+ OpenShot Video Editor (http://www.openshot.org), an open-source project
+ dedicated to delivering high quality video editing and animation solutions
+ to the world.
+
+.. OpenShot Video Editor is free software: you can redistribute it and/or 
+ modify  it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+.. OpenShot Video Editor is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+.. You should have received a copy of the GNU General Public License
+ along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
+
+.. Edited September 27, 2020 - [USA]TechDude
+
+User Manual
+===================
+Thank you for your interest in OpenShot Video Editor, the easy to use, quick to 
```

Comment:
I would prefer we stick to a consistent tag line... and use it in the 3 to 5 places we repeat it (vs rewording it each time). 

Thank you for your interest in OpenShot Video Editor, the award-winning, free and open-source .... etc...